rotkehlchen/data_migrations/upgrades/upgrade_1.py

```diff
@@ -0,0 +1,42 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.typing import Location
+
+if TYPE_CHECKING:
+    from rotkehlchen.rotkehlchen import Rotkehlchen
+
+
+def data_upgrade_1(rotki: 'Rotkehlchen') -> None:
```

Comment:
Here everywhere /s/upgrade/migration

Also the directory should not be upgrades but migrations.

Consistency!!!